backend/btrixcloud/orgs.py

```diff
@@ -909,17 +911,16 @@ async def import_org(
         # the reset password workflow using their email address after import.
         for user in stream_org.get("userDetails", []):  # type: ignore
             try:
-                user_res = await self.user_manager.create_non_super_user(
+                new_user = await self.user_manager.create_user(
                     email=user["email"],
-                    password=secrets.token_hex(20),
```

Comment:
FYI @tw4l removing this, as `create_user()` now supports generating password using the generate_password() function automatically: https://github.com/webrecorder/browsertrix/pull/1902/files#diff-f96501d659f627e51e2c733e9b65807eaba43f6169130a0ae34cd5983f8c0cd0R334